components/raftstore-v2/src/bootstrap.rs

```diff
@@ -71,7 +83,9 @@ impl<'a, ER: RaftEngine> Bootstrap<'a, ER> {
         Ok(Some(ident.get_store_id()))
     }
 
-    fn inner_bootstrap_store(&mut self) -> Result<u64> {
+    /// Allocates a store ID from PD and bootstraps the engine with it. Returns
+    /// the store ID if success.
+    fn bootstrap_store_imp(&mut self) -> Result<u64> {
```

Comment:
seems unnecessary to extract a impl

Reply:
It's renamed from `inner_bootstrap_store` to `bootstrap_store_imp`.

Reply:
I know, but it can be merged into `bootstrap_store` directly